### PR TITLE
Enable prettier in ci

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
Extracted from: https://github.com/embroider-build/embroider/pull/1384
since the editorconfig tooling may be suspicious (or misconfigured)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,11 @@
     "clean": "git clean -x -f",
     "compile": "tsc",
     "lint": "concurrently 'yarn:lint:*'",
+    "lint:fix": "concurrently 'npm:lint:*:fix' --names 'fix:'",
```
Comment (void-mAlex) [inline package.json:33]: any reason to have these called automatically by the lint command? it doesn't seem to exclude :fix as the default ember bp does

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+/test-packages/*/dist/
+/test-packages/*/tmp/
+
+# dependencies
+bower_components/
+node_modules/
+
+# misc
+coverage/
+!.*
```
Comment (void-mAlex) [inline .prettierignore:51]: I think this is in the default bp but does it actually make sense? isn't this just re-adding all files?

**Review by void-mAlex — COMMENTED** [review]
what's the motivation to change quotes in template/html files?
as far as I can tell default in ember is double quotes in templates single for js files

File: .prettierignore (hunk 2 above)
Comment (NullVoxPopuli) [reply .prettierignore:51]: it should only be re-adding `.`-prefixed files, like `.eslintrc.js`

**Comment by NullVoxPopuli** [comment]
> what's the motivation to change quotes in template/html files?
as far as I can tell default in ember is double quotes in templates single for js files

I didn't change the prettier config -- this PR only hooks up prettier to C.I. and package.json#scripts. -- if there is a better config to use, lemme know.

File: packages/util/tests/dummy/app/templates/application.hbs [hunk 3]
```diff
@@ -1,3 +1,3 @@
-<h2 id="title">Welcome to Ember</h2>
```
Comment (mansona) [inline packages/util/tests/dummy/app/templates/application.hbs:1]: I feel like this shouldn't be changed 🤔 because it's not the same as a default output

**Review by mansona — CHANGES_REQUESTED** [review]
we discussed this in the meeting

File: packages/util/tests/dummy/app/templates/application.hbs (hunk 3 above)
Comment (NullVoxPopuli) [reply packages/util/tests/dummy/app/templates/application.hbs:1]: resolved!

Comment (NullVoxPopuli) [reply packages/util/tests/dummy/app/templates/application.hbs]: resolved for real now

**Review by mansona — APPROVED** [review]
Looks good to me 👍

**Review by void-mAlex — APPROVED** [review]

**Pull request merged**